package task_day5;
//
//public class Main {

	//public static void main(String[] args) {
	//try {
			//int[] myNumbers= {1,2,3};
			//System.out.println(myNumbers[10]);
		//	}
			//catch(Exception e) {
			//	System.out.println("error cleared");
			//finally {
			//	System.out.println("The'try catch' is finished");
			//}
		//	}
			
	//}

//}

//        (try,catch,finally)

public class Main {

	public static void main(String[] args) {
	try {
			int[] myNumbers= {1,2,3};
			System.out.println(myNumbers[10]);
			}
			catch(Exception e) {
				System.out.println("error cleared");
			}
			finally {
				System.out.println("The'try catch' is finished");
			}
			}
			
	}


